bin/sfn-compile.py

```diff
@@ -42,9 +42,10 @@
                         help="heaviside file to compile")
 
     args = parser.parse_args()
+    print(args)
```

Comment:
debug print? or we leaving there for user awareness

Reply:
This was a test to see if anyone ever looks at this PR. You passed. 😀

Fixed, thanks.